integrations/http/http.go

```diff
@@ -155,13 +187,11 @@ func (i integration) getConnection(ctx context.Context) map[string]string {
 	return c
 }
 
-func setBody(req *http.Request, rawBody string, formBody map[string]string, jsonBody sdktypes.Value) error {
-	errMutuallyExclusive := errors.New("raw_body, form_body, and json_body are mutually exclusive")
-
-	// Raw body with unknown content type.
-	if rawBody != "" {
-		if formBody != nil || (jsonBody.IsValid() && !jsonBody.IsNothing()) {
-			return errMutuallyExclusive
+func setBody(req *http.Request, bodyType string, rawBody string, formBody map[string]string, jsonBody sdktypes.Value) error {
+	switch bodyType {
+	case bodyTypeRaw:
```

Comment:
this should also be the default

```suggestion
	case "", bodyTypeRaw:
```

Reply:
fixed, in one additional place as well